apis/Google.Cloud.Logging.Console/docs/index.md

```diff
@@ -31,6 +31,34 @@ var builder = WebApplication.CreateBuilder(args);
 builder.Logging.AddGoogleCloudConsole(options => options.IncludeScopes = true);
 ```
 
+## Log trace correlation
+
+The console formatter supports Google Cloud Trace and log correlation. This means that
+if the application using the console formatter is running in Google Cloud and exports traces
+to Google Cloud Trace, the trace context information, if available, can be included in the log entry.
+
+To enable log trace correlation, set the `TraceGoogleCloudProjectId` property of the formatter options
+to the ID of the Google Cloud Project where trace data is being written to Google Cloud Trace.
+This property is optional, and if not set, the trace context information will not be
+included in the log entry. It has no effect on where log entries are written to or
+whether trace information is exported to Google Cloud Trace or to the Google Cloud Project they are exported to.
+
+Note that when application runs in Google Cloud, for instance in Google Cloud Run,
+trace information is automatically collected and exported by the runtime.
+
+If the application is responsible for initiating the traces, it is application's responsibility to ensure
+that the traces are exported in the Google Cloud Trace, in the same Google Cloud Project as specified in the console formatter options.
+
+To enable log trace correlation, the `TraceGoogleCloudProjectId` property of the options should be set to a valid Google Cloud Project ID
```

Comment:
There's a lot of repetition in these docs, this paragrah is almost identialca to the one starting in line 40.